.env_template

```diff
@@ -1,4 +1,6 @@
-POSTGRES_PASSWORD=
-POSTGRES_USER=
-POSTGRES_DB=
-POSTGRES_PORT=
\ No newline at end of file
+export POSTGRES_PASSWORD=
```

Comment:
not my favourite change to make but was made for convenience purposes on local development. This looks different on prod.